models/pixel.js

```diff
@@ -78,7 +78,7 @@ PixelSchema.statics.addPixel = function(colour, x, y, userID, callback) {
     x = parseInt(x), y = parseInt(y);
     if(isNaN(x) || isNaN(y)) return callback(null, { message: "Invalid positions provided." });
     // TODO: Get actual position below:
-    if(x <= 0 || y <= 0 || x > 1000 || y > 1000) return callback(null, { message: "Position is out of bounds." });
+    if(x < 0 || y < 0 || x > 1000 || y > 1000) return callback(null, { message: "Position is out of bounds." });
```

Comment:
Wait, I just noticed Apple had changed this in https://github.com/dynasticdevelop/place/commit/f1a1f2f86b46a10e490b580d5ec6d8972042c921 to fix a critical bug.